docs/cloudmanager-api-overview.md

```
@@ -126,3 +127,59 @@ _Appears in:_
+
+
+EKSKubernetesEngineSpec defines the desired state of EKSKubernetesEngine.
+
+
+
+_Appears in:_
+- [EKSKubernetesEngine](#ekskubernetesengine)
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `dependencies` _[Dependencies](#dependencies)_ | Dependencies defines the dependency configurations for the EKS Kubernetes Engine. |  |  |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)